Composer/packages/lib/shared/src/types/shell.ts

```diff
@@ -16,6 +18,11 @@ export interface BotSchemas {
   diagnostics?: any[];
 }
 
+export interface ExternalUpdate {
```

Comment:
Can you put a little bit of the design and intention here into comments maybe.

I know you are tracking some data is updated outside externally to Form or Visual.